### PR TITLE
Issue 1388

### PR DESCRIPTION
**Pull request by ciaranlp**
Issue url: https://github.com/adaptlearning/adapt_framework/issues/1388

adding 'skip navigation' button to template

### CODE REVIEW

**Comment by oliverfoster** [comment]
should also need this in the src/core/templates in adapt framework https://github.com/ciaranlp/adapt_framework/blob/0a7856b5d728618c4b996831b58f116440e1c389/src/core/templates/navigation.hbs

File: templates/navigation.hbs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {{! Maintainers - Daryl Hedley}}
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link" aria-label="Skip navigation" data-event="skipNavigation" role="button">Skip Navigation</button>
```
Comment (moloko) [inline templates/navigation.hbs]: shouldn't the aria-label be in course.json to allow it to be changed/translated?

Comment (oliverfoster) [inline templates/navigation.hbs]: indeed it should

Comment (oliverfoster) [inline templates/navigation.hbs]: could you change line 5 to:
```
<button class="skip-nav-link" aria-label="{{_globals._accessibility._ariaLabels.skipNavigation}}" data-event="skipNavigation" role="button">Skip Navigation</button>
```

and add _globals._accessibility._ariaLabels.skipNavigation = "Skip Navigation" to the appropriate schema + course.json?

Comment (tomgreenfield) [inline templates/navigation.hbs]: And please remove `role="button"`.

**Comment by ciaranlp** [comment]
Hey guys I've update the hbs as per suggestions

File: templates/navigation.hbs [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {{! Maintainers - Daryl Hedley}}
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link a11y-ignore a11y-ignore-focus" data-event="skipNavigation" tabindex="0" aria-label="{{_globals._accessibility._ariaLabels.skipNavigation}}">{{{_globals._accessibility.skipNavigationText}}}</button>
```
Comment (tomgreenfield) [inline templates/navigation.hbs]: Please remove `tabindex="0"` that seems to have crept in and we should be good to go.

Comment (ciaranlp) [inline templates/navigation.hbs]: thats the tabindex gone @tomgreenfield

**Pull request merged**